docs/core/docker/publish-as-container.md

```diff
@@ -87,7 +87,16 @@ The worker template loops indefinitely. Use the cancel command <kbd>Ctrl+C</kbd>
 
 ## Add NuGet package
 
-The [Microsoft.NET.Build.Containers](https://www.nuget.org/packages/Microsoft.NET.Build.Containers) NuGet package is currently required to publish non-web projects as a container. To add the `Microsoft.NET.Build.Containers` NuGet package to the worker template, run the following [dotnet add package](../tools/dotnet-add-package.md) command:
+Starting with .NET SDK version 8.0.200, the `PublishContainer` target is available for every project. To avoid depending on the `Microsoft.NET.Build.Containers` NuGet package, ensure that you're using the latest .NET SDK version. Additionally, your project file needs to have `IsPublishable` set to `true` and enable SDK container support. To enable SDK container support, set the `EnableSdkContainerBuildSupport` property to `true` in your project file.
```

Comment:
two notes:
* IsPublishable is true by default for console/webapp/worker applications, so the majority of users will not have to specify this
* `EnableSdkContainerSupport` is the property (slight typo on your end :))  and it's set by default for Web and Worker projects - only Console projects need to add this property

Reply:
Updated, thank you for the review and corrections, much appreciated.